common/src/main/java/com/microsoft/identity/common/internal/cache/BrokerOAuth2TokenCache.java

```diff
@@ -1668,26 +1684,9 @@ private static MicrosoftFamilyOAuth2TokenCache initializeFociCache(@NonNull fina
                 );
     }
 
-    /**
-     * Returns the TokenCache to use for supplied client and environment.
-     *
-     * @param clientId          The target client id.
-     * @param environment       The target environment.
-     * @param callingProcessUid The uid of the calling process.
-     * @return The {@link MsalOAuth2TokenCache} matching the supplied criteria or null, if no matching
-     * cache was found.
-     */
     @Nullable
-    private MsalOAuth2TokenCache getTokenCacheForClient(@NonNull final String clientId,
-                                                        @NonNull final String environment,
-                                                        final int callingProcessUid) {
-        final String methodName = ":getTokenCacheForClient";
-
-        final BrokerApplicationMetadata metadata = mApplicationMetadataCache.getMetadata(
-                clientId,
-                environment,
-                callingProcessUid
-        );
+    private MsalOAuth2TokenCache getTokenCacheForClient(@Nullable final BrokerApplicationMetadata metadata) {
+        final String methodName = ":getTokenCacheForClient (1 arg)";
```

Comment:
seems like an attempt to distinguish an overload....how do we disntiguish the following?

```java
 private void methodName(TypeA input)
```

```java
 private void methodName(TypeB input)
```

Reply:
Made less ambiguous in ceea53560b6510d9d363a9fbc49c7dff5812da74